app/src/main/java/com/cornellappdev/coffee_chats_android/fragments/EditInterestsGroupsFragment.kt

```diff
@@ -37,6 +38,7 @@ class EditInterestsGroupsFragment : Fragment(), OnFilledOutObservable {
     private lateinit var selectedItemsAdapter: UserFieldAdapter
     private lateinit var moreItemsAdapter: UserFieldAdapter
 
+
```

Comment:
Nit: Don't include whitespace changes in PR unless they're ones introduced by the autoformatter.